runtime/gc_vlhgc/CopyForwardCompactGroup.hpp

```diff
@@ -50,6 +57,7 @@ class MM_CopyForwardCompactGroup
 	MM_LightweightNonReentrantLock *_copyCacheLock; /**< The lock associated with the list this copy cache belongs to */
 	void *_TLHRemainderBase;  /**< base pointers of the last unused TLH copy cache, that might be reused  on next copy refresh */
 	void *_TLHRemainderTop;	  /**< top pointers of the last unused TLH copy cache, that might be reused  on next copy refresh */
+	uintptr_t _gcIDForTLHRemainder;  /**< the gc cycle when preserving the TLHRemainder. */
```

Comment:
while in this file, convert existing fields to use uintptr_t instead of UDATA etc.